styleguide/source/_patterns/05-pages/news.json

```diff
@@ -233,37 +225,36 @@
     ],
     "learnMore": {
       "heading": {
-        "level": 5,
-        "class": "ama__h5",
         "text": "Learn More About:"
       },
       "links": [
         {
           "title": "Link to something",
           "href": "#",
           "text": "Lorem ipsum dolor sit amet.",
-          "target": "_self"
+          "target": "_self",
+          "class": "ama__link--black"
         },
         {
           "title": "Link to something",
           "href": "#",
           "text": "Lorem ipsum dolor sit amet.",
-          "target": "_self"
+          "target": "_self",
+          "class": "ama__link--black"
         },
         {
           "title": "Link to something",
           "href": "#",
           "text": "Lorem ipsum dolor sit amet.",
-          "target": "_self"
+          "target": "_self",
+          "class": "ama__link--black"
```

Comment:
This should probably not be added back?

Reply:
do you want me to use the @extends here instead of adding the class back?

Reply:
or do you think it's not necessary?

Reply:
I agree with your initial thoughts that adding the extends would be out of scope. I just don't want to see any changes on this relative to link classes or not.